packages/ffe-form/less/phone-number.less

```diff
@@ -20,8 +20,6 @@
 // Styleguide ffe-form.phone-number
 
 .ffe-phone-number {
-    display: flex;
-
```

Comment:
Jeg er litt usikker om denne endring burde være en egen commit. Henger den sammen med den breaking change i ffe-forms-react?
Jeg har aldri sett dobbel-scope i commit-meldingen, dvs de to pakkenavn i `feat(ffe-form, ffe-form-react)`. Fungerer det med conventional-commits?

Reply:
Det vet jeg egentlig ikke hvordan vil fungere i praksis, her henger disse veldig tett sammen for at stylingen skal bli riktig.

Så i praksis vil vi at dagens versjon av PhoneNumber skal bruke dagens versjon av ffe-form, mens den nye versjonen må også ha den nye versjonen av ffe-form. Hvordan gjøres dette best i praksis i designsystemet?

Reply:
Tror det blir best med én PR med to BREAKING CHANGE commits - en på ffe-from og en på ffe-form-react. Det er da så vi pleier å håndtere slik situasjoner i ffe-pakkene som du beskriver.

Reply:
Da er den splittet i to :)